gemrb/core/Scriptable/Actor.cpp

```diff
@@ -9273,10 +9273,14 @@ bool Actor::UseItem(ieDword slot, ieDword header, const Scriptable* target, ieDw
 			// ignore timestop
 			pro->TFlags |= PTF_TIMELESS;
 		}
+	}
+
+	if (pro) {
 		attackProjectile = pro;
-	} else { // launch it now as we are not attacking
-		GetCurrentArea()->AddProjectile(pro, Pos, tar->GetGlobalID(), false);
+		attackProjectile->MoveTo(tar->GetCurrentArea(), ranged ? Pos : tar->Pos);
+		attackProjectile->SetTarget(tar->GetGlobalID(), false);
 	}
+
```

Comment:
This chunk is very suspicious. For fake launches the attack projectile is pointing to freed memory.